parachain/src/chain_spec/kintsugi.rs

```diff
@@ -255,7 +255,7 @@ fn kintsugi_mainnet_genesis(
             inflation: FixedU128::checked_from_rational(2, 100).unwrap(), // 2%
         },
         polkadot_xcm: kintsugi_runtime::PolkadotXcmConfig {
-            safe_xcm_version: Some(2),
+            safe_xcm_version: Some(3),
```

Comment:
Is this a noteworthy change? Why is it only now required?

Reply:
It's required for the benchmarks. Since this is part of genesis it doesn't affect mainnet